### DISCUSSION TITLE
Rendere domotico un pulsante

### DISCUSSION THREAD

**Discussion by Cupra1979** [indomus/forum#346] [Hardware & Firmware]
Ciao a tutti!

In garage ho un pulsante che utilizzo per aprire o chiudere un cancello automatico. Il cancello è motorizzato FAAC con telecomando rolling code ed a suo tempo ho fatto montare appunto questo pulsante per poterlo aprire/chiudere senza telecomando. Il pulsante è un Bticino serie Living Now modello K4005 (1P, N/O) che dà quindi solo l'impulso per apertura, chiusura o "stop" se premuto a cancello in movimento.

Cosa posso usare per rendere tale pulsante domotico, visto che il pulsante Living Now già domotico non esiste (a differenza degli interruttori)? Può fare al mio caso un SONOFF ZBMINI Zigbee? Viste le dimensioni, entrerebbe senza problemi nella scatola (una 503 con 2 frutti occupati).

Grazie mille per i consigli!!

**Comment by indomus** [comment]
Ti basta un qualsiasi attuatore a contatto pulito https://indomus.it/progetti/domotizzare-qualsiasi-telecomando-anche-rolling-code-tramite-contatto-pulito/

**Reply by Cupra1979** [reply]
Grazie, avevo trovato quell'articolo ma mi era rimasto il dubbio che si potesse applicare anche sull'attuatore ZigBee.
Vedo se riesco a trovarne uno per fare qualche prova.

**Reply by indomus** [reply]
Occhio che semmai lo devi modificare a contatto pulito, ovviamente

**Reply by Cupra1979** [reply]
Si, infatti sto cercando online qualche risorsa che parli del contatto pulito su quel modello specifico. Mi piacerebbe poter restare sullo ZigBee come protocollo, per quanto possibile.

**Reply by angelopampalone** [reply]
> Occhio che semmai lo devi modificare a contatto pulito, ovviamente

Scusate se entro in una discussione in cui non sono coinvolto, ma, sempre chiara squadra di @indomus ; visto quanti lo chiedono sul forum, non potreste preparare una guida su come rendere  a contatto pulito sonoff e shellies e espxxx ecc ecc?
